README.md

```diff
@@ -6,3 +6,14 @@ This library only supports Intel Processor Trace, but in the future we hope to
 support alternatives such as Arm's CoreSight.
 
 **This is experimental code.**
+
+## Building
+
+Building is performed by Cargo. In short, most users will want:
+
+```
+cargo build --release
+```
+
+You can set `WITH_SYSTEM_IPT=...` to specify a path to a system libipt to use.
```

Comment:
Although it's a bit subjective `WITH_SYSTEM_` feels like a weird prefix. Why "with"? Why does the "system" have to have an ipt install? I think what you're really saying is "if you want to use your own version of libibt (whether or not it's installed system wide or not) set this var". How about just `IPT_PATH` or similar?

Reply:
It's standard wording for build systems, e.g. in tex live we have:

```
			--with-system-freetype2 \
			--with-system-gd \
			--with-system-libpng \
			--with-system-ncurses \
			--with-system-pnglib \
			--with-system-zlib \
...
```

"system" means the one provided elsewhere, usually the package manager.

Reply:
It's a standard for configure switches, I agree, but not the environment variables. e.g. we just `CC` `CXX` and so on.

Reply:
How about `IPT_PREFIX`. That actually better conveys that I expect there to be `include` and `lib` subdirs.

Reply:
It's better than what's before, but `PREFIX` makes me think of the part of a filename before a "."...